CHANGELOG.md

```diff
@@ -2,25 +2,25 @@
 
 ## Next Release
 ### Internal
+- [In Progress: 30 Nov 2020] Migrate app to use ViewBinding
+- [In progress: 01 Dec 2020] Migrate `ShortCodeSearchResultScreen` to Mobius
+- Add PR comment commands for running instrumented tests and rebasing PR
```

Comment:
Can you recheck the rebase operation, please? It doesn't match the current `DEMO` PR.

https://github.com/simpledotorg/simple-android/blob/release/2020-12-14/CHANGELOG.md

Reply:
Hey, I have added the Demo release things match in the Demo section. I added the rest of the new changes in the Next release section.

Reply:
Oh, wait, yeah this correct. My bad